src/services/socket.js

```diff
@@ -1,7 +1,7 @@
 import io from 'socket.io-client';
 import { dispatch } from '../store/store';
 import { setCurrentWord, setStatus, setRoundNumber, setCurrentPlayer, setPlayers, setCurrentCards, setSocket } from '../store/actions';
-import { setPlayWordAndCard, setPlayCard, setPlayedCard } from '../store/playerActions';
+import { setPlayWordAndCard, setPlayCard, setPlayedCard, setVoteCard, setVotedCard, playWord, resetFinishRound } from '../store/playerActions';
```

Comment:
Forgot to change this but you can change this to `import { types } from '../store/playerActions` and then use `types.THE_ACTION_TYPE` everywhere

Reply:
I think cause we're importing the actual functions, not the types, they need to stay separate?

Reply:
Oh yeah I forgot these weren't the action types